lib/adapters/sql.ex

```diff
@@ -181,7 +181,7 @@ defmodule Apartmentex.Adapters.SQL do
   end
 
   defp query(repo, sql, params, outer_queue_time, mapper, opts) do
-    {pool_mod, pool, timeout} = repo.__pool__
+    {pool_mod, pool, timeout, _} = repo.__pool__
```

Comment:
I'll try to dig into it tonight, but probably something that doesn't matter.. sorry for the delayed responses, I've been overseas and then on a road trip ;-)